src/replica/replica.h

```diff
@@ -284,7 +284,7 @@ class replica : public serverlet<replica>, public ref_counter, public replica_ba
     //
     // Statistics
     //
-    void update_commit_qps(int count);
+    METRIC_DECLARE_INCREMENT_BY(committed_requests, );
```

Comment:
It's strange too to pass such style of parameters.

Reply:
OK, `committed_requests` will be moved to replica-level and maintained in `replication_app_base`, which is more reasonable.